lab-steven/test/auth-routes-test.js

```diff
@@ -0,0 +1,108 @@
+'use strict';
+
+const chai = require('chai');
+const http = require('chai-http');
+chai.use(http);
+const expect = chai.expect;
+const User = require('../model/user.js');
+const server = require('../server');
+const Promise = require('bluebird');
+const mongoose = require('mongoose');
+mongoose.Promise = Promise;
+
+const testUser = {
+  username: 'test-un',
+  email: 'test@email.com',
+  password: 'test-pw',
+};
+
+describe('auth-routes.js', function(){
+
+  describe('#POST /api/signup', function(){
+    afterEach(function(done){
+      User.remove({})
```

Comment:
good clean up on your db. this is good practice so all your tests use fresh datum.